Cloud + Enterprise 0.25.3 Documentation (#400)
astronomer/docs: cloud/stable/05_manage-astronomer/01_workspace-permissions.md

If you do _not_ specify a role in this command, `WORKSPACE_VIEWER` will be set by default. In all cases where a user is invited to a Workspace and deployment-level role is not specified, no deployment-level role will be assumed.

> #### Via Houston API

Reviewer: ```suggestion
#### via Houston API
```

Not sure if this just be upper or lower-case, but should be consistent with the H4's above

Author: Changing the ones above!

> Users with Workspace Admin privileges can use Astronomer's Houston API to programmatically invite users to a Workspace. Within the mutation query, you can configure a user's permissions, login email address, and whether they need to verify their email before accessing the platform. For an example API request, see [Sample Mutations](/docs/cloud/stable/manage-astronomer/houston-api#sample-mutations).

Reviewer: ```suggestion
Users with Workspace Admin privileges can use Astronomer's Houston API to programmatically invite users to a Workspace. By running a variation of the `workspaceAddUser` mutation, you can configure a user's permissions, login email address, and whether they need to verify their email before accessing the platform. For an example API request, see [Sample Mutations](/docs/cloud/stable/manage-astronomer/houston-api#sample-mutations).
```

Reviewer: I wouldn't say "within the mutation query" - in graphql speak it's either a mutation or it's a query (in this case a mutation since it's changing a value in the DB) 

https://www.apollographql.com/blog/graphql/basics/mutation-vs-query-when-to-use-graphql-mutation/

> Users with Workspace Admin privileges can use Houston API mutations to programmatically invite users to a Workspace. Within the mutation query, they can configure a user's permissions, their login information, and whether they need to first verify their email before accessing the platform. For examples of adding Workspace users via mutation, see [Sample Mutations](/docs/cloud/stable/manage-astronomer/houston-api#sample-mutations).

Reviewer 2: "their login information" -> "their login email address" would be more accurate

> Users with Workspace Admin privileges can use Houston API mutations to programmatically invite users to a Workspace. Within the mutation query, they can configure a user's permissions, their login information, and whether they need to first verify their email before accessing the platform. For examples of adding Workspace users via mutation, see [Sample Mutations](/docs/cloud/stable/manage-astronomer/houston-api#sample-mutations).

Reviewer: ```suggestion
Users with Workspace Admin privileges can use Astronomer's Houston API to programmatically invite users to a Workspace. Within the mutation query, you can configure a user's permissions, login email address, and whether or not they need to verify their email before accessing the platform. For an example API request, see [Sample Mutations](/docs/cloud/stable/manage-astronomer/houston-api#sample-mutations).
```

You don't really "use" mutations

> 

Reviewer 3: @aliotta please review this doc change

Workspace _Admins_ are the highest-tiered role at the Workspace level. Admins:

- Can manage users and their permissions in a Workspace.
- Can perform CRUD (create, read, update, delete) operations on the Workspace (e.g. delete the Workspace, change its name).
- Can create Airflow Deployments in the Workspace.
- Can perform CRUD operations on any Airflow Deployment within the Workspace.
- Can manage **Billing**.
- Can perform CRUD operations on any Service Account in the Workspace.

> 

Reviewer 3: This change looks good. @bote795  - please provide a final review.

Reviewer 4: Lgtm

> Workspace _Admins_ do not automatically have CRUD access to all Airflow Deployments within it - they must either create or be added to those Deployments and will be held to the restrictions of their deployment-level role. For example, a Workspace _Admin_ could be a Deployment _Viewer_ and not have access to push code to that Deployment.

Reviewer 4: this needs to be changed admins now automatically have CRUD to all airflow deployments within it without explicit access
